integrations/observability/prometheus-operator/README.md

```diff
@@ -0,0 +1,53 @@
+# Observability Integration: Prometheus Operator
+
+This integration aims to mimic a common pattern which is to use the Prometheus Operator
+to monitor a Kubernetes cluster. In this integration there is an OpenTelemetry Collector
+that receives OpenTelemetry metrics fom the KubeArchive components and then exposes
+them using a Prometheus Exporter. Then a `ServiceMonitor` resource instructs the Prometheus
+Operator to scrap the metrics from the OpenTelemetry Collector.
+
+**Note**: traces are sent to the OpenTelemetry collector but they don't get forwarded
+anywhere. This is done to avoid that KubeArchive's components log errors about not being
+able to send traces.
+
+To visualize metrics forward the Grafana UI, visit http://localhost:3000 and log in with
```

Comment:
In later parts of this README you use `port-forward` instead of just `forward`

```suggestion
To visualize metrics port-forward the Grafana UI, visit http://localhost:3000 and log in with
```

Reply:
Ops! Thanks, fixed.